profiler/src/ProfilerEngine/Datadog.Profiler.Native.Linux/CrashReportingLinux.cpp

```diff
@@ -339,11 +339,28 @@ std::vector<int32_t> CrashReportingLinux::GetThreads()
             if (entry->d_name[0] == '.')
                 continue;
             auto threadId = atoi(entry->d_name);
-            threads.push_back(threadId);
+            auto threadName = GetThreadName(threadId);
+            threads.push_back(std::make_pair(threadId, threadName));
         }
 
         closedir(proc_dir);
     }
 
     return threads;
+}
+
+std::string CrashReportingLinux::GetThreadName(int32_t tid)
```

Comment:
I would reuse OpSysTools::GetNativeThreadName() instead

Reply:
It gets the name of the thread for another process